### DISCUSSION TITLE
Let's talk about https://github.com/serverless-heaven/serverless-webpack/pull/451

### DISCUSSION THREAD

**Discussion by designfrontier** [serverless-heaven/serverless-webpack#1537] [Team Posts]
So #451 is the kind of breaking change that I am generally opposed to. But before I say that over there I wanted to have a discussion with y'all about it. 

It clarifies things... but would break config for what is likely a huge percentage of the user base. If y'all are agreed I am happy to lead that conversation out on the patchset.

**Comment by HyperBrain** [comment]
Fully agree. This is a fundamental _breaking_ change. In my opinion we must not change the default - if we did, we had to introduce the change with version 6.0.0. It forces nearly everyone to check his projects.
I'd propose to decline the PR.

**Comment by designfrontier** [comment]
closed and commented
